tests/e2e/spec/advanced.spec.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+import { browser, expect, $ } from '@wdio/globals';
+import {
+  disableCache,
+  enableExtension,
+  getExtensionElement,
+  getExtensionPageURL,
+  setToggle,
+  switchToPanel,
+} from '../utils.js';
+
+import { PAGE_DOMAIN, PAGE_URL } from '../wdio.conf.js';
+
+async function setPrivacyToggle(name, value) {
+  await browser.url(await getExtensionPageURL('settings'));
+  await setToggle(name, value);
+}
+
+async function setCustomFilters(filters) {
+  await setPrivacyToggle('custom-filters', true);
+
+  const input = await getExtensionElement('input:custom-filters');
+  await input.setValue(filters.join('\n'));
+
+  const checkbox = await getExtensionElement(
+    'checkbox:custom-filters:trusted-scriptlets',
+  );
+
+  if (!(await checkbox.getProperty('checked'))) {
+    await checkbox.click();
+    await expect(checkbox).toBeChecked();
+  }
+
+  await getExtensionElement('button:custom-filters:save').click();
+  await expect(
+    getExtensionElement('component:custom-filters:result'),
+  ).toBeDisplayed();
+}
+
+describe('Advanced Features', function () {
+  before(enableExtension);
+
+  describe('Custom Filters', function () {
+    before(async function () {
+      await setCustomFilters([
+        `@@connect.facebook.net^`,
+        `${PAGE_DOMAIN}###custom-filter`,
+        `${PAGE_DOMAIN}##+js(rpnt, h1, Test Page, "Hello world")`,
+      ]);
+
+      await disableCache();
```

Comment:
Can we hide this away? Cannot we set it one time before all tests?

Reply:
I thought about it too, there is the`browser.overwriteCommand` API - we can add disabling cache on Chrome before we call the original `browser.url`.